addons/mail/static/src/components/chat_window_header/chat_window_header.xml

```diff
@@ -15,63 +15,63 @@
                         record="chatWindowHeaderView.threadIconView"
                     />
                 </t>
-                <div class="o_ChatWindowHeader_item o_ChatWindowHeader_name mh-100 mx-1 my-0 text-truncate" t-att-class="{'ms-3':!chatWindow.thread or chatWindow.thread.model !== 'mail.channel'}" t-att-title="chatWindow.name">
-                    <t t-esc="chatWindow.name"/>
+                <div class="o_ChatWindowHeader_item o_ChatWindowHeader_name mh-100 mx-1 my-0 text-truncate" t-att-class="{'ms-3':!chatWindowHeaderView.chatWindowOwner.thread or chatWindowHeaderView.chatWindowOwner.thread.model !== 'mail.channel'}" t-att-title="chatWindowHeaderView.chatWindowOwner.name">
```

Comment:
by the way you could replace `chatWindowHeaderView.chatWindowOwner.thread.model !== 'mail.channel'` by `!chatWindowHeaderView.chatWindowOwner.thread.channel`

Reply:
done